fearless/Common/Helpers/AmountFormatterFactory.swift

```diff
@@ -79,53 +84,104 @@ struct AmountFormatterFactory: NumberFormatterFactoryProtocol {
     // swiftlint:disable function_body_length
     private func createCompoundFormatter(
         for preferredPrecision: Int,
-        roundingMode: NumberFormatter.RoundingMode = .down
+        roundingMode: NumberFormatter.RoundingMode = .down,
+        for locale: FormatterLocale
     ) -> LocalizableDecimalFormatting {
-        let abbreviations: [BigNumberAbbreviation] = [
-            BigNumberAbbreviation(
-                threshold: 0,
-                divisor: 1.0,
-                suffix: "",
-                formatter: DynamicPrecisionFormatter(
-                    preferredPrecision: UInt8(preferredPrecision),
-                    roundingMode: roundingMode
+        let abbreviations: [BigNumberAbbreviation]
+        switch locale {
+        case .japanese:
+            abbreviations = [
```

Comment:
It is advisable create struct with abbreviations arrays in other file
struct Abbreviations {
static let japanese = []
static let usual = []
}